public/src/jQuery.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function() {
+
+  bowling = new Bowling();
+
+  $('#throwBall').click(function() {
+    var score = $('#scoreChoosen').val();
+    if ( bowling.points.length <= 19) {
+      bowling.throw_record(parseInt(score));
+      writePtAndScore();
+      writeSpareBonus();
+      bowling.increaseActualFrame();
+    } else if (bowling.points.length === 20) {
+      bowling.throw_record(parseInt(score));
```

Comment:
We went through this a bit in the Monday code review, but whenever you find yourself using if/else for game rules or business logic, ask whether you might get better results via Polymorphism or Strategy patterns. See:

https://refactoring.guru/replace-conditional-with-polymorphism